components/relevancy/src/lib.rs

```diff
@@ -47,9 +57,53 @@ impl RelevancyStore {
     ///
     ///  This method may execute for a long time and should only be called from a worker thread.
     #[handle_error(Error)]
-    pub fn ingest(&self, _top_urls_by_frecency: Vec<String>) -> ApiResult<()> {
+    pub fn ingest(&self, top_urls_by_frecency: Vec<String>) -> ApiResult<HashMap<Interest, u32>> {
```

Comment:
For now, I just have the frequency of the number of interest occurrences, but we would probs start discussing which method we want to use to express interest?

Reply:
That makes a lot of sense to me.

Reply:
Lets take in a Vec<Struct> instead of string

Reply:
lets move this logic to a separate module and call it `classify()`